content/en/docs/reference/config-api/kubelet-config.v1beta1.md

```diff
@@ -943,7 +943,7 @@ Default: &quot;&quot;</p>
 <td>
    <p>systemReservedCgroup helps the kubelet identify absolute name of top level CGroup used
 to enforce <code>systemReserved</code> compute resource reservation for OS system daemons.
-Refer to <a href="https://git.k8s.io/design-proposals-archive/node/node-allocatable.md">Node Allocatable</a>
+Refer to <a href="https://git.k8s.io/community/contributors/design-proposals/node/node-allocatable.md">Node Allocatable</a>
```

Comment:
The target page of this link is 404 Not Found.

Reply:
Please propose a fix in the upstream source [https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubelet/config/v1beta1/types.go#L628].

Reply:
I opened a PR: https://github.com/kubernetes/kubernetes/pull/111997 .